package.xml

```diff
@@ -5,7 +5,7 @@
   <description>CMake scripts for building Arduino Leonardo (atmega32u4) Firmwares</description>
 
   <maintainer email="mpurvis@clearpathrobotics.com">Mike Purvis</maintainer>
-  <license>BSD</license>
+  <license>MPL, BSD</license>
```

Comment:
It should be multiple tags: https://www.ros.org/reps/rep-0140.html#license-multiple-but-at-least-one

Reply:
Fixed.